sources/install.sh

```diff
@@ -1337,15 +1337,15 @@ function install_pwnedornot() {
   add-test-command "pwnedornot.py --help"
 }
 
+function install_chromium() {
+  fapt chromium
+  add-test-command "chromium --version"
+}
+
 function install_ghunt() {
```

Comment:
Could you find a test command for ghunt ? To validate the chromium dependencies etc

Reply:
The installation of Ghunt is commented as this tool has problems but I added a test-command for gowitness to validate the chromium dependencies

Reply:
Maybe add a FIXME comment then

Reply:
ok it's done